packages/wallet/src/redux/protocols/reducer-helpers.ts

```diff
@@ -321,3 +323,18 @@ export function getFundingChannelId(channelId: string, sharedData: SharedData):
     return getFundingChannelId(channelIdToCheck, sharedData);
   }
 }
+
+export function removeZeroFundsFromBalance(
+  incomingAllocation: string[],
+  incomingDestination: string[],
+): { allocation: string[]; destination: string[] } {
+  const allocation: string[] = [];
+  const destination: string[] = [];
+  for (let i = 0; i < incomingAllocation.length; i++) {
```

Comment:
This could be something like
```
incomingAllocation.map((a, i) => {
  if (bigNumberify(a).gt(0)) {
      allocation.push(incomingAllocation[i]);
      destination.push(incomingDestination[i]);
  }
}
```